src/Symfony/Component/PropertyInfo/Tests/Extractor/PhpDocExtractorTest.php

```diff
@@ -352,3 +368,15 @@ public function setOmittedType(array $omittedTagType)
     {
     }
 }
+
+class UnionTypeDocBlock
+{
+    /**
+     * Here we use an union type, as commonly used in PhpDoc, to make sure it is ignored and does not trigegr an exception.
```

Comment:
typo: s/trigegr/trigger

Reply:
Good spot! And actually that code is useless now, I'm removing it.